spec/convertkit/configuration_spec.rb

```diff
@@ -1,22 +0,0 @@
-require "spec_helper"
```

Comment:
I've removed this file, because the exactly same thing is tested in `spec/convertkit/convertkit_spec.rb`.